### PR TITLE
[FAST_BUILD] Clean conda env

### PR DESCRIPTION
**Pull request by mathbunnyru**
## Describe your changes

Introduce the env variable which environment to activate by default.
This makes the custom env recipe smaller.
Also eliminates the problem of double activation in the case of a custom environment (if you call `bash` a few times, you will get a `base, custom, base, custom, base, custom...` chain).

## Checklist (especially for first-time contributors)

- [ ] I have performed a self-review of my code
- [ ] If it is a core feature, I have added thorough tests
- [ ] I will try not to use force-push to make the review process easier for reviewers
- [ ] I have updated the documentation for significant changes

### CODE REVIEW

File: docs/using/recipe_code/custom_environment.dockerfile [hunk 1]
```diff
@@ -33,16 +33,8 @@ RUN "${CONDA_DIR}/envs/${env_name}/bin/pip" install --no-cache-dir \
...
-# RUN \
-#     # This changes a startup hook, which will activate the custom environment for the process
-#     echo conda activate "${env_name}" >> /usr/local/bin/before-notebook.d/10activate-conda-env.sh && \
-#     # This makes the custom environment default in Jupyter Terminals for all users which might be created later
-#     echo conda activate "${env_name}" >> /etc/skel/.bashrc && \
-#     # This makes the custom environment default in Jupyter Terminals for already existing NB_USER
-#     echo conda activate "${env_name}" >> "/home/${NB_USER}/.bashrc"
-
-USER ${NB_UID}
+ENV DOCKER_STACKS_CONDA_ENV "${env_name}}"
```
Comment (yuvipanda) [inline docs/using/recipe_code/custom_environment.dockerfile:40]: An extra `}`?

Comment (mathbunnyru) [reply docs/using/recipe_code/custom_environment.dockerfile]: Thanks, missed it.

**Comment by mathbunnyru** [comment]
With this PR we will also be close to not hardcoding PATH.
This should also help custom envs - they will be able not to accidentally consume binaries from base environment.
I don't think I will do it, but if someone wants to, I can explain what to do.

**Comment by mathbunnyru** [comment]
@yuvipanda @benz0li @consideRatio please tell me, what you think about this.

**Comment by mathbunnyru** [comment]
I think this creates additional complexity to most users, when they don't use custom environments.
And I haven't seen any complaints / issues with the current implementation of custom environments, so let's close this PR.

**Pull request closed without merging**